_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -100,10 +100,21 @@ This `input` element is not [focusable][] because of the `disabled` attribute.
 This `a` element is not [focusable][] because it moves focus to the `input` element whenever it receives focus.
 
 ```html
+<div id="sampleModal" role="dialog" aria-label="Sample Modal" aria-modal="true" style="border: solid black 1px; padding: 1rem;">
+    <label>Some input <input id="dialogFirst"></label><br />
+    <button id="closeButton">Close button</button>
```

Comment:
Should we have a more real example, e.g. "input your name" or something like that. "Some input" doesn't sound like a good label name (which this rule doesn't check, but it really add no value to the example to have placeholder text).

Reply:
Will change to "First and last name"